tmcprototype/SdpSubarrayLeafNode/SdpSubarrayLeafNode/SdpSubarrayLeafNode.py

```diff
@@ -80,33 +82,17 @@ def commandCallback(self, event):
     # -----------------
     # Device Properties
     # -----------------
-
-
-
-
-
-
     SdpSubarrayNodeFQDN = device_property(
-        dtype='str', default_value=CONST.PROP_DEF_VAL_TM_MID_SDP_SA,
+        dtype='str', default_value="mid_sdp/elt/subarray_1",
```

Comment:
Can be replaced with CONST.<<string variable>>

Reply:
ReadTheDocs is not working when CONST file variable is used. Therefore kept it as a string for now.